src/account/SecuritySubmenu.tsx

```diff
@@ -0,0 +1,532 @@
+          details={t('shareAnalytics_detail')}
+        />
+        <SettingsExpandedItem
+          title={t('removeAccountTitle')}
+          details={t('removeAccountDetails')}
+          onPress={onRemoveAccountPress}
+          testID="ResetAccount"
+        />
+        <SettingsExpandedItem
+          title={t('deleteAccountTitle')}
+          details={t('deleteAccountDetails')}
+          onPress={onDeleteAccountPress}
+          testID="DeleteAccount"
+        />
+        {getDevSettingsComp()}
+        <Dialog
+          isVisible={!!promptConfirmRemovalModal}
+          title={t('promptConfirmRemovalModal.header')}
+          actionText={t('promptConfirmRemovalModal.resetNow')}
+          actionPress={confirmAccountRemoval}
+          secondaryActionText={t('cancel')}
+          secondaryActionPress={hideConfirmRemovalModal}
+          testID="ConfirmAccountRemovalModal"
+        >
+          {t('promptConfirmRemovalModal.body')}
+        </Dialog>
+      </ScrollView>
+      <Toast
+        withBackdrop
+        variant={NotificationVariant.Warning}
+        description={t('keylessBackupSettingsDeleteError')}
+        showToast={showDeleteKeylessBackupError}
+        onPressCta={onDismissKeylessBackupError}
+        onUnmount={onDismissKeylessBackupError}
+        onDismiss={onDismissKeylessBackupError}
+        ctaLabel={t('dismiss')}
+        title={t('error')}
+        testID="KeylessBackupDeleteError"
+      />
+      <BottomSheet
+        forwardedRef={resetAccountBottomSheetRef}
+        title={t('accountKeyModal.header')}
+        description={`${t('accountKeyModal.body1')}\n\n${t('accountKeyModal.body2')}`}
+        testId="ResetAccountBottomSheet"
+      >
+        <Button
+          style={{ marginTop: Spacing.Regular16 }}
+          size={BtnSizes.FULL}
+          type={BtnTypes.SECONDARY}
+          onPress={onPressContinueWithAccountRemoval}
+          text={t('continue')}
+          disabled={revokeNumberAsync.loading}
+          testID="ResetAccountButton"
+        />
+      </BottomSheet>
+      <BottomSheet
+        forwardedRef={deleteAccountBottomSheetRef}
+        title={t('deleteAccountWarning.title')}
+        description={t('deleteAccountWarning.description')}
+        testId="DeleteAccountBottomSheet"
+      >
+        <Button
+          style={styles.bottomSheetButton}
+          size={BtnSizes.FULL}
+          type={BtnTypes.SECONDARY}
+          onPress={handleDeleteAccount}
+          text={
+            revokeNumberAsync.loading
+              ? t('deleteAccountWarning.buttonLabelRevokingPhoneNumber')
+              : t('deleteAccountWarning.buttonLabel')
+          }
+          disabled={revokeNumberAsync.loading}
+          testID="DeleteAccountButton"
+        />
+      </BottomSheet>
+    </SafeAreaView>
+  )
+}
+
+const styles = StyleSheet.create({
+  header: {
+    paddingHorizontal: variables.contentPadding,
+  },
+  title: {
+    ...typeScale.labelSemiBoldMedium,
+    margin: 16,
```

Comment:
can we use the `Spacing` enum? there are several other magic numbers in the styles below